packages/core/analytics/core-analytics-browser-internal/src/track_performance_measure_entries.ts

```diff
@@ -0,0 +1,57 @@
+/*
```

Comment:
I don't have a strong opinion on that. I initially located the file in this directory mainly because the other trackers(click, viewport)  also exist here but it also make sense to me to co-locate the observer with the components in the `packages/kbn-ebt-tools/src/performance_metrics/*`